src/tools/sfdk/command.cpp

```diff
@@ -1136,10 +1136,19 @@ Worker::ExitStatus BuiltinWorker::runEmulator(const QStringList &arguments, int
     }
 
     if (arguments.first() == "start") {
-        if (arguments.count() > 2) {
-            qerr() << P::unexpectedArgumentMessage(arguments.at(2)) << endl;
+        QCommandLineParser parser;
+        QCommandLineOption headlessOption("headless");
+        parser.addOptions({headlessOption});
+
+        if (!parser.parse(arguments)) {
+            qerr() << parser.errorText() << endl;
             return BadUsage;
         }
+
+        if (parser.isSet(headlessOption)) {
+            emulator->virtualMachine()->setHeadless(true);
```

Comment:
Of course I checked before using setHeadless() that it's not saved. You originally said that you wanted "--headless" option, now there is one. It does what is needed for the use case. Please give me better reasoning than just "I think it will be better" if you want me to scrap all the work I already did and start over from scratch!

Reply:
I see that 1) completing the implementation of --headless options would require a considerable amount of work while 2) the need to use these options throughout the test code would not be as convenient as having the possibility to set it on one place, which 3) can be implemented with very little effort, so I am certain we should...

Reply:
In the meanwhile I figured that this is just too much work when the test suite can be just run in xvfb - one line of change in the script.